style.css

```diff
@@ -8,185 +8,546 @@ html {
+  color: #b3bac5;
+}
+
+.button {
+  width: 14.2rem;
+  height: 4.8rem;
+  padding: 1.2rem 1.6rem;
+  background: #fff;
+  border: none;
+  border-radius: 0.8rem;
+  font-family: Poppins, sans-serif;
+  font-weight: 500;
+  font-size: 1.7rem;
+  text-align: center;
+  line-height: 2.4rem;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  transition: all 0.4s;
+  cursor: pointer;
+}
+
+.button:hover {
+  opacity: 0.5;
+}
+
+@media (max-width: 500px) {
+  .header {
+    width: 37.5rem;
+  }
+
   .text {
-    background: url("images/Header\ bg.svg");
-    background-size: cover;
+    width: 37.5rem;
+    height: 60.4rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 3rem 0;
+  }
+
+  .text-section {
+    width: 32.7rem;
+  }
+
+  .work {
+    width: 37.5rem;
+    padding: 11.4rem 2.4rem;
+  }
+
+  .card-work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1.6rem;
+    gap: 1.2rem;
+    border: 1px solid #dfe1e6;
+    border-radius: 1.6rem;
+  }
+
+  .snapshot-portfolio {
+    width: 29.5rem;
+    height: 22rem;
+    padding: 0;
+  }
+
+  .snapshot-portfolio img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  .left-block {
     display: flex;
     flex-direction: column;
     align-items: flex-start;
-    padding: 11.4rem 2.4rem;
+    padding: 0;
     gap: 1.2rem;
-    position: absolute;
-    height: 60.4rem;
-    left: calc(50% - 92rem / 2);
-    top: 9.2rem;
+    width: 29.5rem;
   }
 
-  .head-section {
-    width: 92rem;
-    height: 15rem;
-    font-family: Poppins, sans-serif;
-    font-style: normal;
-    font-weight: 700;
+  .work-title {
     font-size: 4rem;
-    line-height: 5.2rem;
+    line-height: 4.4rem;
+    color: #172b4d;
+  }
+
+  .frame {
+    font-size: 1.3rem;
+  }
+
+  .para {
+    font-size: 1.5rem;
+  }
+
+  .about {
     display: flex;
+    flex-direction: column;
     align-items: center;
-    color: #172b4d;
-    flex: none;
-    order: 0;
-    flex-grow: 0;
-  }
-
-  .para-section {
-    width: 92rem;
-    height: 14.6rem;
-    font-family: Poppins, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 1.6rem;
-    line-height: 2.4rem;
-    color: #344563;
-    flex: none;
-    order: 1;
-    flex-grow: 0;
+    width: 37.5rem;
+    background-color: #fff;
+    gap: 2.4rem;
+    border-top-right-radius: 10rem;
+  }
+
+  .about-text-section {
+    width: 32.7rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1.2rem;
+  }
+
+  .skill {
+    width: 32.7rem;
+  }
+
+  .languages ul {
+    display: flex;
+    flex-direction: column;
+    gap: 1.2rem;
+    list-style: none;
+  }
+
+  .languages ul li {
+    width: 30.3rem;
+    height: 6.672rem;
+    padding-left: 7rem;
+    display: flex;
+    align-items: center;
+  }
+
+  .languages ul li img {
+    position: absolute;
+    left: 1.2rem;
+    top: 50%;
+    transform: translateY(-50%);
+  }
```

Comment:
- [ ] The alignment of language pictures and text is still not correct as the previous reviewer mentioned. I think instead of targeting these images,`ul` and `li` tags and styling them individually, you can just give all `li` the same class and target that class.
that class will have `display: flex;` and will align its two children perfectly.

your design
![asdf](https://user-images.githubusercontent.com/96683843/212465520-d2d3c5c1-c25b-42c4-ae11-d98cb50e21f8.jpg)


figma design
![as](https://user-images.githubusercontent.com/96683843/212465505-29ade275-e47c-4959-aa39-91d18bdf83a6.jpg)